src/daemons/StorageDaemon.cpp

```diff
@@ -72,23 +75,37 @@ int main(int argc, char *argv[]) {
     }
 
     if (FLAGS_data_path.empty()) {
-        LOG(FATAL) << "Storage Data Path should not empty";
-        return -1;
+        LOG(ERROR) << "Storage Data Path should not empty";
+        return EXIT_FAILURE;
+    }
+
+    auto result = nebula::network::NetworkUtils::getLocalIP(FLAGS_local_ip);
+    if (!result.ok()) {
+        LOG(ERROR) << "Get localIp failed, ip " << FLAGS_local_ip
+                   << ", status:" << result.status();
+        return EXIT_FAILURE;
+    }
+    uint32_t localIP;
+    if (!NetworkUtils::ipv4ToInt(result.value(), localIP)) {
+        LOG(ERROR) << "Convert ip failed, ipstr:" << result.value();
+        return EXIT_FAILURE;
     }
-    LOG(INFO) << "Starting the storage Daemon on port " << FLAGS_port
-              << ", dataPath " << FLAGS_data_path;
 
+    if (FLAGS_meta_server_addrs.empty()) {
```

Comment:
We should add a regular to check format of FLAGS_meta_server_addrs？

Reply:
Good proposal.  We could check the format of server_address inside NetWorkUtils::toHosts

Reply:
FYI. gflags supports some kind of *validator* that will be check against whenever a specific flag is being modified. We could utilize that at the first place for most of our flags. We could do that in future.